scripts/poweron/power-on.py

```diff
@@ -41,18 +41,14 @@ def main(session, args):
     
     power_on_config = session.xenapi.host.get_power_on_config(remote_host)
     
-    if mode == "iLO" or mode=="DRAC" :
+    if mode=="DRAC" :
```

Comment:
What if someone had this configured to iLO previously? Should we log something here?